hedgehog/src/Hedgehog/Gen.hs

```diff
@@ -768,6 +769,16 @@ bytes range =
 ------------------------------------------------------------------------
 -- Combinators - Choice
 
+-- | Trivial generator that always produces the same element.
+--
+-- All of the following expressions are equivalent:
+--
+-- * @'constant' a@
+-- * @'pure' a@
+-- * @'element' [a]@
+constant :: Monad m => a -> Gen m a
+constant = pure
```

Comment:
I should write up the style I'm following in this repo so I can just point to it, but this needs to be:
```hs
constant :: Monad m => a -> Gen m a
constant =
  pure
```